uitools/widgets/Esri/ArcGISRuntime/Toolkit/BasemapGallery.ui

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>BasemapGallery</class>
+ <widget class="QFrame" name="BasemapGallery">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>400</width>
+    <height>300</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>Frame</string>
```

Comment:
Vis-a-vis the grey border around your ListView. In Qt designer, you should be able to set your layout's margins to `0, 0, 0, 0` to remove the border.

Reply:
Thanks for that, working as intended